lib/vintage_net_wizard/web/router.ex

```diff
@@ -16,10 +16,16 @@ defmodule VintageNetWizard.Web.Router do
   plug(:dispatch)
 
   get "/" do
-    render_page(conn, "index.html",
-      configs: Backend.configurations(),
-      format_security: &display_security_from_key_mgmt/1
-    )
+    case Backend.configurations() do
+      [] ->
+        redirect(conn, "/networks")
```

Comment:
Does this need to be `networks.html`?

Reply:
No, the `/networks` route will handle that.